Creating the Tern, Ibis, and Petrel (#4546)
endless-sky/endless-sky: data/remnant ships.txt

ship "Ibis"
	sprite "ship/ibis"
	thumbnail "thumbnail/ibis"
	attributes
		category "Medium Warship"
		licenses
			"Remnant Capital"

> 		"cost" 30000000

Reviewer: This is 50% more expensive than the hull cost of the Albatross. Seems a bit much given that this is a medium warship.

Author: Reduced to a bit lower than the hull price of an Albatross.

		"shields" 22000

> 		"hull" 11000

Reviewer: These shield and hull values seem pretty round.

Author: Um. I like round numbers? Not sure if this actually needs fixing.

Reviewer: >I like round numbers?

Factions outside of Republic space don't tend to have round numbers. :)

Author: OK, I'll break them up a bit.

		"required crew" 20
		"bunks" 48

> 		"mass" 300

Reviewer: This seems rather light, especially when you compare it to the heavier Pelican that is worse in HP by quite a bit (33k vs 17k). I'd say reduce the health of this carrier to be closer to that of the Pelican.

Author: Reduced both Shields and hull a bit, and upped the mass by 50 so its hull mass is now 30 higher than the Pelican, rather than 20 smaller.

		"drag" 7.1
		"heat dissipation" 0.7
		"fuel capacity" 700
		"ramscoop" 2
		"outfit space" 568

> 		"cargo space" 103

Reviewer: Outfit space and cargo space positions are flipped.

Author: Fixed.

> 		"weapon capacity" 200

Reviewer: *Exactly* 200? As a medium warship and a carrier, I'd expect less weapon space (and maybe more engine space instead of the exact same as the Albatross).

Author: Weapon Space reduced to 180, and engine space increased to 160. How's that look?

Reviewer: As with above, factions outside Republic space don't tend to have such "nice" numbers. Fudge them up or down a bit.

Author: OK. Made them not nice.

		"engine capacity" 142
		"shield generation" 6
		"shield energy" 4.6
		"hull repair rate" 2

> 		"hull energy" 1.2

Reviewer: More shield and hull generation than the Albatross, but at the same energy drain?

Author: Brought the numbers in much closer to the Albatrosses. Dropped all the values a little, since it isn't a heavy warship and doesn't have as much to regenerate. That being said, it is going to be a little more energy efficient. Not by much, though.

		"outfit scan power" 18
		"outfit scan speed" 1
		"tactical scan power" 39
		weapon
			"blast radius" 240
			"shield damage" 2400
			"hull damage" 1200
			"hit force" 3000
	outfits
		"Inhibitor Cannon" 3

> 		"Thrasher Turret" 1

Reviewer: You don't need to specify `1`.

Author: Removed the 1

		"Point Defense Turret" 2

		"Aeon Cell"
		"Crystal Capacitor" 3
		"Emergency Ramscoop" 2
		"Thermoelectric Cooler" 5
		"Quantum Key Stone"
		"Salvage Scanner"
		"Tuning Rifle" 20

		"Forge-Class Thruster"
		"Crucible-Class Thruster"
		"Forge-Class Steering"
		"Crucible-Class Steering"
		"Scram Drive"
		
	drone 0 30 over 
	drone -34 68 over
	drone 0 81 over
	drone 34 68 over
	fighter -25 10 under
	fighter 25 10 under
	engine -16.5 120 0.7
	engine 0 116.5 1
	engine 16.5 120 0.7
	gun 0 -135.5 "Inhibitor Cannon"
	gun 15.5 -102 "Inhibitor Cannon"
	gun -15.5 -102 "Inhibitor Cannon"
	turret 0 -62 "Thrasher Turret"
	turret 0 -20.5 "Point Defense Turret"
	turret 0 58.5 "Point Defense Turret"

> 	

Reviewer: Extra whitespace.

Author: Fixed.

	leak "leak" 50 50
	leak "flame" 50 80
	leak "big leak" 90 30
	explode "tiny explosion" 10
	explode "small explosion" 25
	explode "medium explosion" 25
	explode "large explosion" 10
	"final explode" "final explosion medium"

> 	description "This is the Ibis, a multipurpose carrier that serves as a mining platform for terns, as an escort ship for pelican fleets, and a combat platform for terns used in screening roles. It is capable of fielding a single wing of support craft, consisting of two petrels and four terns."

Reviewer: `This is the <ship>...` isn't the way that any other ship descriptions are worded, instead using `The <ship> is...`
You need to capitalize the names of the other ships (Pelican, Petrel, Tern, Albatross).

Author: OK, changed.

> 	description "	The Ibis was designed shortly after the albatross as a means of extending the range of their resource collection efforts after the Remnant realized that an Alpha invsion might not be as imminent as original thought."

Reviewer: `invsion` -> `invasion`

Author: Thanks. Fixed.

		"hull repair rate" 2
		"hull energy" 1.2
		"outfit scan power" 18
		"outfit scan speed" 1
		"tactical scan power" 39
		weapon
			"blast radius" 240
			"shield damage" 2400
			"hull damage" 1200
			"hit force" 3000
	outfits
		"Inhibitor Cannon" 3
		"Thrasher Turret" 1
		"Point Defense Turret" 2

		"Aeon Cell"
		"Crystal Capacitor"
		"Emergency Ramscoop" 2
		"Thermoelectric Cooler" 3
		"Quantum Key Stone"
		"Salvage Scanner"
		"Tuning Rifle" 20

		"Forge-Class Thruster"
		"Crucible-Class Thruster"
		"Forge-Class Steering"
		"Crucible-Class Steering"
		"Hyperdrive"
		
	drone 0 30 over 
	drone -34 68 over
	drone 0 81 over
	drone 34 68 over
	fighter -25 10 under
	fighter 25 10 under
	engine -16.5 120 0.7
	engine 0 116.5 1
	engine 16.5 120 0.7
	gun 0 -135.5 "Inhibitor Cannon"
	gun 15.5 -102 "Inhibitor Cannon"
	gun -15.5 -102 "Inhibitor Cannon"
	turret 0 -62 "Thrasher Turret"
	turret 0 -20.5 "Point Defense Turret"
	turret 0 58.5 "Point Defense Turret"
	
	leak "leak" 50 50
	leak "flame" 50 80
	leak "big leak" 90 30
	explode "tiny explosion" 10
	explode "small explosion" 25
	explode "medium explosion" 25
	explode "large explosion" 10
	"final explode" "final explosion medium"

> 	description "This is the Ibis. A slightly carrier that serves as a mining platform for terns, as well as an escort ship for pelican fleets and a combat platform for terns used in screening roles. Stats are currently made up from whole cloth and need revision."

Reviewer 2: ...A slightly *what* carrier?

Author: Fixed. Thanks!

			"hit force" 3000
	outfits
		"Inhibitor Cannon" 3
		"Point Defense Turret" 2

		"Aeon Cell"
		"Crystal Capacitor" 3
		"Emergency Ramscoop" 2
		"Thermoelectric Cooler" 2
		"Quantum Key Stone"
		"Salvage Scanner"
		"Tuning Rifle" 20

		"Smelter-Class Thruster"
		"Smelter-Class Steering"
		"Forge-Class Steering"
		"Scram Drive"
		
	drone 0 30 over 
	drone -34 68 over
	drone 0 81 over
	drone 34 68 over
	fighter -25 10 under
	fighter 25 10 under
	engine -16.5 120 0.7
	engine 0 116.5 1
	engine 16.5 120 0.7
	gun 0 -135.5 "Inhibitor Cannon"
	gun 15.5 -102 "Inhibitor Cannon"
	gun -15.5 -102 "Inhibitor Cannon"
	turret 0 -62 "Point Defense Turret"
	turret 0 -20.5
	turret 0 58.5 "Point Defense Turret"
	leak "leak" 50 50
	leak "flame" 50 80
	leak "big leak" 90 30
	explode "tiny explosion" 10
	explode "small explosion" 25
	explode "medium explosion" 25
	explode "large explosion" 10
	"final explode" "final explosion medium"
	description "The Ibis is a multipurpose carrier that serves as a mobile mining platform, an escort ship for Pelican fleets, and a combat platform using Terns for screening. It is capable of fielding a single wing of support craft, consisting of two Petrels and four Terns."

> 	description "	The Ibis was designed shortly after the Albatross as a means of extending the range of resource collection efforts after the Remnant realized that an Alpha invasion might not be as imminent as originally believed."

Reviewer 3: `as a means`, is this supposed to be plural?

Reviewer 4: Yes, this is what is called a stock expression, in 'as a means'  the word means is always plural.
(Source: am an English major)

Reviewer 3: yep, resolved

ship "Petrel"
	sprite "ship/petrel"
	thumbnail "thumbnail/petrel"
	attributes
		category "Fighter"
		licenses
			Remnant
		"cost" 4500000
		"shields" 2000
		"hull" 2500
		"cargo space" 20
		"fuel capacity" 100
		"ramscoop" 1
		"required crew" 1
		"bunks" 3
		"mass" 50
		"drag" 1.2
		"heat dissipation" .9
		"outfit space" 195
		"shield generation" 1.9
		"shield energy" 1.1
		"hull repair rate" 0.9
		"hull energy" 0.6
		"outfit scan power" 12
		"outfit scan speed" 1
		"tactical scan power" 26
		"asteroid scan power" 30
		"weapon capacity" 80

> 		"engine capacity" 55

Reviewer: Weapon capacity and engine capacity positions out of order.

Author: Fixed.

Reviewer: No, the order was fine, but these should be up with outfit space.

Author: OK, moved them up with the outfit space.

		weapon
			"blast radius" 15
			"shield damage" 150
			"hull damage" 200
			"hit force" 350
	outfits
		"Inhibitor Cannon" 3
		
		"Millennium Cell" 2
		"Crystal Capacitor"
		"Thermoelectric Cooler" 3
		
		"Crucible-Class Thruster"
		"Crucible-Class Steering"
		
	gun 0 -38.5 "Inhibitor Cannon"
	gun -10.5 -24 "Inhibitor Cannon"
	gun 10.5 -24 "Inhibitor Cannon"
	turret 0 2.5
	engine 0 41.5
	explode "tiny explosion" 30
	explode "small explosion" 20
	description "The Petrel is a multi-role mining/wing leader fightercraft designed to give remnant a forward control position for flights of Terns in higher risk or quickly changing environments. Their initial design was used as a urban mobile gun platform that flitted around structures, caverns, and natural terrain on their worlds while fighting on-planet threats."

> 	description "Due to the comparatively small pool the Remnant draw on for pilots and crew they are loath to throw mass waves of people into the line of fire. As such, their fighters are not intended to be up-close glass cannons like most other factions of humanity tend to build. Instead, these ships are highly agile fighters equipped with medium range weaponry designed to snipe enemies from behind cover."

Reviewer 2: Multi-paragraph descriptions should have every paragraph after the first indented with a tab.

Author: Fixed. Thanks!

ship "Petrel"
	sprite "ship/petrel"
	thumbnail "thumbnail/petrel"
	attributes
		category "Fighter"
		licenses
			Remnant
		"cost" 1141000
		"shields" 2300
		"hull" 1000
		"cargo space" 20
		"fuel capacity" 100
		"ramscoop" 1
		"required crew" 2
		"bunks" 3
		"mass" 50
		"drag" 1.4
		"heat dissipation" .9
		"outfit space" 165
		"weapon capacity" 49
		"engine capacity" 34
		"shield generation" 0.85
		"shield energy" 0.55
		"hull repair rate" 0.38
		"hull energy" 0.28
		"outfit scan power" 12
		"outfit scan speed" 1
		"tactical scan power" 26
		"asteroid scan power" 30
		weapon
			"blast radius" 15
			"shield damage" 150
			"hull damage" 200
			"hit force" 350
	outfits
		"Inhibitor Cannon" 3
		
		"Millennium Cell" 2
		"Crystal Capacitor"
		"Thermoelectric Cooler"
		
		"Crucible-Class Thruster"
		"Crucible-Class Steering"
		
	gun 0 -38.5 "Inhibitor Cannon"
	gun -10.5 -24 "Inhibitor Cannon"
	gun 10.5 -24 "Inhibitor Cannon"
	turret 0 2.5
	engine 0 41.5
	explode "tiny explosion" 30
	explode "small explosion" 20

> 	description "The Petrel is a multi-role mining/wing leader fightercraft designed to give remnant a forward control position for flights of Terns in higher risk or quickly changing environments. Their initial design was used as a urban mobile gun platform that flitted around structures, caverns, and natural terrain on their worlds while fighting on-planet threats."

Reviewer 5: ```suggestion
	description "The Petrel is a multi-role mining/wing leader fightercraft designed to provide support for flights of lighter Terns in high risk environments. The ship's design originated as an a urban mobile gun platform that flitted around structures and natural terrain, but proved easy to adapt into a spacecraft."
```

> 	description "	Due to the comparatively small pool the Remnant draw on for pilots and crew they are loath to throw mass waves of people into the line of fire. As such, their fighters are not intended to be up-close glass cannons like most other factions of humanity tend to build. Instead, these ships are highly agile fighters equipped with medium range weaponry designed to snipe enemies from behind cover."

Reviewer 5: ```suggestion
	description "	The Remnant are loath to sacrifice pilots and crew from the comparatively small pool they are drawn from. As such, their fighters are equipped with medium-range weaponry designed to strike enemies from afar."
```

ship "Petrel"
	sprite "ship/petrel"
	thumbnail "thumbnail/petrel"
	attributes
		category "Fighter"
		licenses
			Remnant
		"cost" 1141000
		"shields" 2300
		"hull" 1000
		"cargo space" 20
		"fuel capacity" 100
		"ramscoop" 1
		"required crew" 2
		"bunks" 3
		"mass" 50
		"drag" 1.4
		"heat dissipation" .9
		"outfit space" 165
		"weapon capacity" 49
		"engine capacity" 34
		"shield generation" 0.85
		"shield energy" 0.55
		"hull repair rate" 0.38
		"hull energy" 0.28
		"gaslining" 1
		"outfit scan power" 12
		"outfit scan speed" 1
		"tactical scan power" 26
		"asteroid scan power" 30
		weapon
			"blast radius" 15
			"shield damage" 150
			"hull damage" 200
			"hit force" 350
	outfits
		"Inhibitor Cannon" 3
		
		"Millennium Cell" 2
		"Crystal Capacitor"
		"Thermoelectric Cooler"
		
		"Crucible-Class Thruster"
		"Crucible-Class Steering"
		
	gun 0 -38.5 "Inhibitor Cannon"
	gun -10.5 -24 "Inhibitor Cannon"
	gun 10.5 -24 "Inhibitor Cannon"
	turret 0 2.5
	engine 0 41.5
	explode "tiny explosion" 30
	explode "small explosion" 20

> 	description "The Petrel is a multi-role mining/wing leader fightercraft designed to provide support for flights of lighter Terns in high risk environments. The ship's design originated as an a urban mobile gun platform that flitted around structures and natural terrain, but proved easy to adapt into a spacecraft."

Author: I think "originated as" is a proper way of stating it.

> 	description "The Petrel is a multi-role mining/wing leader fightercraft designed to provide support for flights of lighter Terns in high risk environments. The ship's design originated as an a urban mobile gun platform that flitted around structures and natural terrain, but proved easy to adapt into a spacecraft."

Reviewer 3: ```suggestion
	description "The Petrel is a multi-role mining/wing leader fightercraft designed to provide support for flights of lighter Terns in high-risk environments. The ship's design originated as an a urban mobile gun platform that flitted around structures and natural terrain, but proved easy to adapt into a spacecraft."
```

> 	description "The Petrel is a multi-role mining/wing leader fightercraft designed to provide support for flights of lighter Terns in high risk environments. The ship's design originated as an a urban mobile gun platform that flitted around structures and natural terrain, but proved easy to adapt into a spacecraft."

Reviewer 3: ```suggestion
	description "The Petrel is a multi-role mining/wing leader fightercraft designed to provide support for flights of lighter Terns in high risk environments. The ship's design originated as an urban mobile gun platform that flitted around structures and natural terrain, but proved easy to adapt into a spacecraft."
```

Author: Fixed. Thanks!

> 	description "The Petrel is a multi-role mining/wing leader fightercraft designed to provide support for flights of lighter Terns in high risk environments. The ship's design originated as an a urban mobile gun platform that flitted around structures and natural terrain, but proved easy to adapt into a spacecraft."

Reviewer 3: ```suggestion
	description "The Petrel is a multi-role mining/wing leader fighter craft designed to provide support for flights of lighter Terns in high risk environments. The ship's design originated as an a urban mobile gun platform that flitted around structures and natural terrain, but proved easy to adapt into a spacecraft."
```
i suppose both is valid but i felt inclined to mention it

Author: Thanks for pointing it out. If both are valid, I think I will keep it as-is.

	gun

ship "Tern"
	sprite "ship/tern"
	thumbnail "thumbnail/tern"
	attributes
		category "Drone"
		licenses
			Remnant
		"cost" 2530000
		"shields" 500
		"hull" 2500
		"cargo space" 5
		"fuel capacity" 50
		"energy capacity" 500
		"ramscoop" 0.5
		"automaton" 1
		"mass" 32
		"drag" .8
		"heat dissipation" .9
		"gaslining" 1
		"outfit space" 75
		"shield generation" 0.6
		"shield energy" 0.4
		"hull repair rate" 1.2
		"hull energy" 0.9
		"atmosphere scan" 100
		"outfit scan power" 12
		"outfit scan speed" 1
		"tactical scan power" 26
		"asteroid scan power" 30
		"weapon capacity" 16

> 		"engine capacity" 26

Reviewer: Weapon capacity and engine capacity positions out of order.

Author: Fixed.

Reviewer: The order was fine, they're just in the completely wrong position (should be up below outfit space).

Author: OK. Moved.

		"ramscoop" 1
		"automaton" 1
		"mass" 30
		"drag" .9
		"heat dissipation" .9
		"gaslining" 1
		"outfit space" 111
		"shield generation" 1.8
		"shield energy" 1.1
		"hull repair rate" 0.7
		"hull energy" 0.6
		"energy generation" 1.8
		"heat generation" 3.0
		"atmosphere scan" 100
		"outfit scan power" 12
		"outfit scan speed" 1
		"tactical scan power" 26
		"asteroid scan power" 30
		"weapon capacity" 36
		"engine capacity" 40
		weapon
			"blast radius" 10
			"shield damage" 100
			"hull damage" 150
			"hit force" 250
	outfits
		"Inhibitor Cannon" 2
		
		"Millennium Cell"
		"Crystal Capacitor"
		"Thermoelectric Cooler"
		
		"Crucible-Class Thruster"
		"Crucible-Class Steering"
		
	gun -5 -20 "Inhibitor Cannon"
	gun 5 -20 "Inhibitor Cannon"
	turret 0 -8
	engine -5 20.5
	engine 5 20.5
	explode "tiny explosion" 30
	explode "small explosion" 20
	description "The Tern is a small, hardy drone developed to plunge into the atmosphere of nearby gas giants to collect valuable resources for the nearby Remnant worlds. As a result, these tough, thick shelled craft are surprisingly effective at providing distraction and covering fire when enemies intrude on Remnant space."

> 	description "Unlike most empires, the Remnant do not have sufficient people to pilot fighter wings in waves likely to sustain significant casualties. As such, they frequently use these hardy drones to fulfill similar screening roles in their home systems, allowing manned craft to focus on more specialized tasks."

Reviewer 2: This one too.

Author: Thank you. Fixed.

	gun

ship "Tern"
	sprite "ship/tern"
	thumbnail "thumbnail/tern"
	attributes
		category "Drone"
		licenses
			Remnant
		"cost" 3830000
		"shields" 1000
		"hull" 4000
		"cargo space" 10
		"fuel capacity" 100
		"ramscoop" 1
		"automaton" 1
		"mass" 30
		"drag" .9
		"heat dissipation" .9
		"gaslining" 1
		"outfit space" 111
		"shield generation" 1.8
		"shield energy" 1.1
		"hull repair rate" 0.7
		"hull energy" 0.6
		"energy generation" 1.8
		"heat generation" 3.0
		"atmosphere scan" 100
		"outfit scan power" 12
		"outfit scan speed" 1
		"tactical scan power" 26
		"asteroid scan power" 30
		"weapon capacity" 36
		"engine capacity" 40
		weapon
			"blast radius" 10
			"shield damage" 100
			"hull damage" 150
			"hit force" 250
	outfits
		"Inhibitor Cannon" 2
		
		"Millennium Cell"
		"Crystal Capacitor"
		"Thermoelectric Cooler"
		
		"Crucible-Class Thruster"
		"Crucible-Class Steering"
		
	gun -5 -20 "Inhibitor Cannon"
	gun 5 -20 "Inhibitor Cannon"
	turret 0 -8
	engine -5 20.5
	engine 5 20.5
	explode "tiny explosion" 30
	explode "small explosion" 20

> 	description "The Tern is a small, hardy, drone developed to plunge into the atmosphere of nearby gas giants to collect valuable resources for the nearby Remnant worlds. As a result, these tough, thick shelled craft are surprisingly effective at providing distraction and covering fire when enemies intrude on Remnant space."

Reviewer 6: `small, hardy, drone` sounds off to me, I'm no grammer nanna but shouldn't it be:
`small, hardy drone`

Author: I'm inclined to agree with you. It does look better without it.

Fixed.

	gun

ship "Tern"
	sprite "ship/tern"
	thumbnail "thumbnail/tern"
	attributes
		category "Drone"
		licenses
			Remnant
		"cost" 860000
		"shields" 1500
		"hull" 700
		"automaton" 1
		"cargo space" 5
		"mass" 32
		"drag" 1
		"heat dissipation" .9
		"fuel capacity" 50
		"energy capacity" 500
		"ramscoop" 0.5
		"outfit space" 75
		"weapon capacity" 16
		"engine capacity" 26
		"shield generation" 0.6
		"shield energy" 0.4
		"hull repair rate" 0.27
		"hull energy" 0.2
		"gaslining" 1
		"atmosphere scan" 100
		"outfit scan power" 12
		"outfit scan speed" 1
		"tactical scan power" 26
		"asteroid scan power" 30
		weapon
			"blast radius" 10
			"shield damage" 100
			"hull damage" 150
			"hit force" 250
	outfits
		"Inhibitor Cannon"
		
		"Millennium Cell"
		"Thermoelectric Cooler"
		
		"Anvil-Class Engine"
		
	gun 0 -22 "Inhibitor Cannon"
	engine -5 20.5
	engine 5 20.5
	explode "tiny explosion" 30
	explode "small explosion" 20

> 	description "The Tern is a small, hardy drone developed to plunge into the atmosphere of nearby gas giants to collect valuable resources for the nearby Remnant worlds. As a result, these tough, thick shelled craft are surprisingly effective at providing distraction and covering fire when enemies intrude on Remnant space."

Reviewer 5: ```suggestion
	description "The Tern is a small, hardy drone developed to plunge into the atmosphere of nearby gas giants for collecting valuable resources. These thick-shelled craft are also surprisingly effective when pressed into service providing distraction and covering fire when enemies intrude on Remnant space."
```

	gun

ship "Tern"
	sprite "ship/tern"
	thumbnail "thumbnail/tern"
	attributes
		category "Drone"
		licenses
			Remnant
		"cost" 860000
		"shields" 500
		"hull" 2500
		"automaton" 1
		"cargo space" 5
		"mass" 32
		"drag" 1.0
		"heat dissipation" .9
		"fuel capacity" 50
		"energy capacity" 500
		"ramscoop" 0.5
		"outfit space" 75
		"weapon capacity" 16
		"engine capacity" 26
		"shield generation" 0.3
		"shield energy" 0.2
		"hull repair rate" 0.6
		"hull energy" 0.4
		"gaslining" 1
		"atmosphere scan" 100
		"outfit scan power" 12
		"outfit scan speed" 1
		"tactical scan power" 26
		"asteroid scan power" 30
		weapon
			"blast radius" 10
			"shield damage" 100
			"hull damage" 150
			"hit force" 250
	outfits
		"Inhibitor Cannon"
		
		"Millennium Cell"
		"Thermoelectric Cooler"
		
		"Anvil-Class Engine"
		
	gun 0 -22 "Inhibitor Cannon"
	engine -5 20.5
	engine 5 20.5
	explode "tiny explosion" 30
	explode "small explosion" 20

> 	description "The Tern is a small, hardy drone developed to plunge into the atmosphere of nearby gas giants for collecting valuable resources. These thick-shelled craft are also surprisingly effective when pressed into service providing distraction and covering fire when enemies intrude on Remnant space."

Author: Hmmm, `intrude in` would work better.

> 	description "The Tern is a small, hardy drone developed to plunge into the atmosphere of nearby gas giants for collecting valuable resources. These thick-shelled craft are also surprisingly effective when pressed into service providing distraction and covering fire when enemies intrude on Remnant space."

Author: Replaced part of that sentence, so it is now:
`These thick-shelled craft are also surprisingly effective when pressed into service as screening ships when enemies intrude into Remnant space.`

I think it sounds better now.

> 	description "	Unlike most empires, the Remnant do not have sufficient people to pilot fighter wings in waves likely to sustain significant casualties. As such, they frequently use these hardy drones to fulfill similar screening roles in their home systems, allowing manned craft to focus on more specialized tasks."

Reviewer 3: sufficient people -> sufficient amounts of people ?

Author: changed "sufficient" to "enough," as that sounds a bit better.

> 	description "	Unlike most empires, the Remnant do not have sufficient people to pilot fighter wings in waves likely to sustain significant casualties. As such, they frequently use these hardy drones to fulfill similar screening roles in their home systems, allowing manned craft to focus on more specialized tasks."

Author: Heh. Empire seemed as good a term as anything. The Coalition feels like an empire, the Unfettered are trying to make one, the Korath *had* one and squandered it. Humanity always likes to make them..

Calling them "factions" feels like I'm almost breaking the fourth wall, but it is probably the next most reasonable term. Or "Nation."

> 	description "	Unlike most empires, the Remnant do not have sufficient people to pilot fighter wings in waves likely to sustain significant casualties. As such, they frequently use these hardy drones to fulfill similar screening roles in their home systems, allowing manned craft to focus on more specialized tasks."

Reviewer 2: I think "spacefaring nations" would be sufficiently generic without feeling meta.

> 	description "	Unlike most empires, the Remnant do not have sufficient people to pilot fighter wings in waves likely to sustain significant casualties. As such, they frequently use these hardy drones to fulfill similar screening roles in their home systems, allowing manned craft to focus on more specialized tasks."

Author: I like that. Changing it.
